ejb/src/test/java/org/wildfly/httpclient/ejb/SimpleInvocationTestCase.java

```diff
@@ -258,10 +266,13 @@ public void testSimpleFailedInvocation() throws Exception {
         }
     }
 
+    /*
+     * TODO: review the idea behind the affinity in this case, test may be invalid
+     */
     @Test
+    @Ignore
```

Comment:
The same here, we either need a follow up Jira or we need to know if there will be a follow up PR for this same Jira (so I can know if I will mark the Jira as resolved after this PR is merged or not).

Reply:
This was something to look at once the basic affinity management is working.